scripts/build_torch_wheels.sh

```diff
@@ -293,6 +311,12 @@ function install_torchaudio_from_source() {
   python setup.py bdist_wheel
   pip install dist/*.whl
   popd
+
+  # collect_wheels expects this
+  if [ ! -d "/pytorch/audio/dist" ]; then
+    mkdir /pytorch/audio/dist
+    cp audio/dist/*.whl /pytorch/audio/dist/
```

Comment:
I thought we already have script to do that.

Reply:
check https://github.com/pytorch/xla/blob/master/docker/common.sh#L49

Reply:
Yea, that actually assumes that the wheel is already placed under `/pytorch/...` https://github.com/pytorch/xla/blob/570c6d44871175785793ab79601b7cb9e3f16146/docker/common.sh#L27.

Reply:
hmm, why wouldn't that be the case for this pr? Wheel generally be placed under `dist/`

Reply:
It's under /pytorch/dist, if pytorch is installed at the root /. It would fail if it's installed and build from else where, since the build script uses relative paths!

Reply:
ok, sg